client/gutenberg/extensions/email-subscribe/editor.scss

```diff
@@ -0,0 +1,47 @@
+.jetpack-email-subscribe-email {
+	display: block;
+	padding: 3px 9px;
+	width: 50%;
+}
+
+.jetpack-email-subscribe-consent-label {
+	color: #999;
+	display: block;
+	font-style: italic;
+	line-height: 20px;
+	margin-bottom: 23px;
+}
+
+.jetpack-email-subscribe-submit {
+	margin: 18px 0 13px;
+}
+
+.jetpack-email-subscribe-form-error {
+	border: 1px solid #ff0000;
+}
+
+.jetpack-email-subscribe-processing {
+	background-color: #e0e5e9;
+	display:none;
+	margin-bottom: 23px;
+	padding: 20px;
+	text-align: center;
+}
+
+.jetpack-email-subscribe-error {
+	background-color: #da5544;
```

Comment:
Let's use these colours from colour variables as we do in Calypso codebase; if these are not Calypso colours but Gutenberg ones, it'll be also easier to migrate pulling them directly from Gutenberg if we can have them as variables named the same.

See example from:
https://github.com/Automattic/wp-calypso/blob/c1043f9dd3e6c6c025e9a9115e68f2d154ea9efd/client/gutenberg/extensions/markdown/editor.scss

Reply:
This is a server side rendered block and this file is only for purposes of emulating the view on the frontend from the file https://github.com/Automattic/jetpack/pull/10597/files#diff-a664f71634843e5599131c9eb846e10a which wp-calypso cannot pull.
I opted for keeping those 2 files in sync.